### PR TITLE
refactor(drizzle): deprecate use of drizzle-nest

### PR DESCRIPTION
**Pull request by G0maa**
deprecate use of drizzle-nest

### CODE REVIEW

**Comment by G0maa** [comment]
uh oh my own ci failed

File: apps/api/schema.graphql [hunk 1]
```diff
@@ -3,5 +3,6 @@
 # ------------------------------------------------------
 
 type Query {
+  db: String!
```
Comment (bassiounix) [inline apps/api/schema.graphql:6]: what is the purpose of this?

File: apps/api/src/app.service.ts
Comment (bassiounix) [inline apps/api/src/app.service.ts:1]: high level services is bad, please move to a module instead

File: apps/api/src/app.resolver.ts
Comment (bassiounix) [inline apps/api/src/app.resolver.ts:1]: remove global modules from global scope into modules folder

File: apps/api/package-lock.json
Comment (alikehel) [inline apps/api/package-lock.json:1]: remove this file.
why did u use npm in the api workspace?
always `pnpm i` only in the root.

**Review by alikehel — CHANGES_REQUESTED** [review]

File: apps/api/schema.graphql (hunk 1 above)
Comment (G0maa) [reply apps/api/schema.graphql:6]: @bassiounix It only exists for making sure that db connection works.

File: apps/api/src/app.service.ts
Comment (G0maa) [reply apps/api/src/app.service.ts:1]: This is used for testing purposes only, will be removed when @abdo0nasser merges his PR.

File: apps/api/src/app.resolver.ts
Comment (G0maa) [reply apps/api/src/app.resolver.ts:1]: same comment above.

**Review by alikehel — APPROVED** [review]

**Pull request merged**